StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -727,8 +752,21 @@ private func CreatePaneViewController(
             viewController = nil
         }
     case .networkingLinkVerification:
-        assertionFailure("Not supported")
-        viewController = nil
+        if let accountholderCustomerEmailAddress = dataManager.manifest.accountholderCustomerEmailAddress {
+            let networkingLinkVerificationDataSource = NetworkingLinkVerificationDataSourceImplementation(
+                accountholderCustomerEmailAddress: accountholderCustomerEmailAddress,
+                manifest: dataManager.manifest,
+                apiClient: dataManager.apiClient,
+                clientSecret: dataManager.clientSecret,
+                analyticsClient: dataManager.analyticsClient
+            )
+            let networkingLinkVerificationViewController = NetworkingLinkVerificationViewController(dataSource: networkingLinkVerificationDataSource)
+            networkingLinkVerificationViewController.delegate = nativeFlowController
+            viewController = networkingLinkVerificationViewController
+        } else {
+            assertionFailure("Code logic error. Missing parameters for \(pane).")
+            viewController = nil
```

Comment:
what should actually happen here? if exactly this (aka showing terminal error) maybe we log a better message?

Reply:
Ideally, we never encounter this as this should never happen (hence the assert, but the assert can be debatable); we have been doing this since the inception of Auth Flow (there's precedence), so LMK on other ideas on how to handle. The thought is that we _think_ this will never happen, but in case it does, if we run in it DEBUG, we can catch it spectacularly failing rather than brushing it aside as "oh, just a random terminal error happened."

And yes, a normal user would see a terminal error:
- https://github.com/stripe/stripe-ios/blob/c4f05f52/StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift#L137-L139

Reply:
makes sense